app/backend/src/couchers/models.py

```diff
@@ -938,6 +938,7 @@ class HostRequest(Base):
     # timezone aware start and end times of the request, can be compared to now()
     start_time = column_property(date_in_timezone(from_date, timezone))
     end_time = column_property(date_in_timezone(to_date, timezone) + text("interval '1 days'"))
+    start_time_to_write_reference = column_property(date_in_timezone(to_date, timezone))
```

Comment:
Document new state. When is this state set?

Reply:
These are expressions that SQLAlchemy will execute in the db when pulling the row. They're not extra state, just some helper expressions. This is because the timezone transformations (that don't yet happen since currently this is in UTC) are easier to do in the db.

Reply:
True, sorry for the noise! Resolving.